src/server/admin/decommission-service/controllers/form.js

```diff
@@ -1,18 +1,32 @@
+import Joi from 'joi'
+
 import { buildOptions } from '~/src/server/common/helpers/options/build-options.js'
 import { fetchRepositories } from '~/src/server/common/helpers/fetch/fetch-repositories.js'
 
 const decommissionFormController = {
+  options: {
+    validate: {
+      query: Joi.object({
+        serviceName: Joi.string().optional()
+      })
+    }
+  },
   handler: async (_request, h) => {
     const { repositories } = await fetchRepositories()
+    const serviceName = _request.query.serviceName ?? ''
 
     const repositoriesValues = repositories.map((repo) => {
       return { value: repo.id, text: repo.id }
     })
 
     const repositoriesOptions = buildOptions(repositoriesValues ?? [])
 
-    return h.view('admin/decommission-service/views/decommission-form', {
+    return h.view('admin/decommission-service/views/form', {
       pageTitle: 'Decommission Service',
+      serviceName,
+      formValues: {
```

Comment:
so `formValues` across the portal codebase is generally used as a context provided via `provideFormContextValues`. If you are provided context here in a view controller I would just send in `serviceName` and then for the input value in the form do `value: serviceName or formValues.serviceName`